Resources/android/ExampleView.js

```diff
@@ -0,0 +1,20 @@
+var View = require('android.view.View'),
+  Activity = require('android.app.Activity'),
+  TiActivity = new Activity(Ti.Android.currentActivity);
+
+exports.create = function(params) {
+  params = params || {};
+
+  var view = new View(TiActivity);
+
+  if (params.backgroundColor) {
+
+    // Hyperloop should provide cross-platform helpers you can use anywhere in
+    // the app to do stuff like this:
+    var gColor = Hyperloop.colorFromString(params.backgroundColor);
+
+    view.setBackgroundColor(gColor);
+  }
```

Comment:
I think we should not influence this too much by how current modules work. It's really more comparable to Alloy widgets (#6). There's nothing a module can do that you couldn't do in the app. It's just about packaging anything you **can** do in an app as a stand-alone package.